.github/workflows/test_docker_build.yaml

```diff
@@ -0,0 +1,71 @@
+name: Test Skyportal Docker Build
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Test SkyPortal Docker build
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+
+    steps:
+      - uses: actions/setup-python@v1
+        with:
+          python-version: "3.8"
+
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache
+            ~/.local
+            ~/.npm
+          key: ${{ runner.os }}-${{ hashFiles('**/lockfiles') }}
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          submodules: true
+
+      - name: Install Docker and Docker Compose
+        run: |
+          sudo apt-get update
+          sudo apt-get install \
+            apt-transport-https \
+            ca-certificates \
+            curl \
+            gnupg-agent \
+            software-properties-common
+          curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+          sudo apt-key fingerprint 0EBFCD88
+          sudo add-apt-repository \
+            "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+            $(lsb_release -cs) \
+            stable"
+          sudo apt-get install docker-ce docker-ce-cli containerd.io
+          docker --version
+
+          sudo curl -L "https://github.com/docker/compose/releases/download/1.28.3/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+          sudo chmod +x /usr/local/bin/docker-compose
+          docker-compose --version
+
+      - name: Build Docker image
+        run: |
+          make docker-local
+
+      - name: Start containers
+        run: |
+          docker-compose up -d
+
+      - name: Create database tables
+        run: |
+          # Give plenty of time for containers to fully spin up
+          sleep 90
+          docker exec skyportal_web_1 bash -c 'source /skyportal_env/bin/activate && PYTHONPATH=$PYTHONPATH:"." python skyportal/initial_setup.py --config=config.yaml --username testuser@cesium-ml.org'
```

Comment:
@acrellin Would know more about the history of initial_setup.py.  I thought we didn't use it anymore.  Would `make db_init` followed by running the data loader not do the trick?

Reply:
Yeah I didn't think we were using `initial_setup.py` so I decided to repurpose it for initial setup of container volumes (hence the updates to the docs as well). The `make db_init` approach will not work here because `db_init.py` (in baselayer) actually only creates the DB but does not involve table creation. That's usually handled by `make run`, but since the Docker images run in production mode, the `make run` will not create the tables either. The data loader as well does not do any table creation as it just makes API calls to upload data.

Reply:
Could initial_setup.py be replaced by the data loader?  If so, it already has some waiting logic built-in (can be improved).

If not, yes, it would be good to find a way to ping the system for being online.  I think you can connect to the migration manager and ask it for status, e.g.

Reply:
The solution I arrived at is to remove the sleep in the workflow and instead add waiting/retry logic into initial_setup.py. I don't think replacing it with the data loader is the best idea as it would involve logic pretty different from what is currently in the data loader. What we want to do is wait for the PostgreSQL database to be up and running and run `create_tables` in it, and not wait for the API to be up as the data loader currently does (as the API will fail to stand up properly until the tables are created, as I described in my other comment). Thoughts?

Reply:
This seems related to https://github.com/skyportal/skyportal/issues/1802#event-4575401691 then

Reply:
@acrellin Any thoughts?

Reply:
I think it'd be better to move away from the otherwise deprecated initial_setup and use the data loader if possible, but I'm not opposed if the latter would be much more complicated

Reply:
Ok sure, let me see if I can rework the data loader to work for this purpose